.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@c29wang can you remove this file/folder?

Reply:
It is removed! Ty!